logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/call/Call.kt

```diff
@@ -18,5 +19,6 @@ data class Call(
     val isCameraOn: Boolean,
     val callerId: String,
     val participants: List<Participant> = emptyList(),
-    val maxParticipants: Int = 0 // Was used for tracking
+    val maxParticipants: Int = 0, // Was used for tracking
+    val activeSpeakers: List<ActiveSpeaker> = emptyList()
```

Comment:
Isn't better if we put active speaker fields in Participant  object ?

Reply:
Great suggestion!